server/services/authService.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
+
+const dbName = "ParkGames";
+const collectionUsers = "Users";
+const collectionUserEvents = "UserEvents";
+const assert = require("assert");
+var ObjectId = require("mongodb").ObjectID;
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
+
+const {
+  getUserByIdRepo,
+  getUserByUserNameRepo,
+  insertNewUserRepo,
+  createUserEventRepo,
+} = require("../repositories/authRepository");
+
+require("dotenv").config();
+
+//------------------SERVICE-----------------------
+// the brains of the application, they control any payload manipulation/validation
+//---------------------------------------------------
+
+//get a user by _id in the user collection.
+const getUserById = async (id) => {
+  const checkForUser = await getUserByIdRepo(id);
+  if (!checkForUser) {
```

Comment:
If you `return` inside the if, you don't need an `else`

```
if (thing) {
   return do thing;
} else {
   return do other thing;
}
```

is the same as 

```
if (thing) {
   return do thing;
} 

return do other thing;
```

Reply:
good to know!